minio/README.md

```diff
@@ -0,0 +1,318 @@
+# Kubeflow
```

Comment:
Please, leave minio/README.md blank (if you want a placeholder) instead of copying the entire Kubeflow installation steps. I think it makes no sense.